CONTRIBUTING.md

```diff
@@ -35,3 +35,9 @@ Must be one of the following:
 - **refactor**: A code change that neither fixes a bug nor adds a feature
 - **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
 - **test**: Adding missing tests or correcting existing tests
+
+### Becoming an EddieHub Maintainer
+
+EddieHub is all about open source collaboration. We want it to be a place which is inclusive and welcoming for all and therefore, our maintainers are carefully chosen from those we see upholding the EddieHub guiding principles of collaboration first, code second, making quality contributions to EddieHub over a longer period of time and those that follow the [CODE of CONDUCT](CODE_OF_CONDUCT.md).
```

Comment:
This is left 😅 

```suggestion
EddieHub is all about open source collaboration. We want it to be a place that is inclusive and welcoming for all and therefore, our maintainers are carefully chosen from those we see upholding the EddieHub guiding principles of collaboration first, code second, making quality contributions to EddieHub over a longer period of time and those that follow the [CODE of CONDUCT](CODE_OF_CONDUCT.md).
```

Reply:
There's a missing comma after `period of time`, I think. That's what Grammarly suggests.

Reply:
```suggestion
EddieHub is all about open source collaboration. We want it to be a place that is inclusive and welcoming for all and therefore, our maintainers are carefully chosen from those we see upholding the EddieHub guiding principles of collaboration first, code second, making quality contributions to EddieHub over a longer period of time, and those that follow the [CODE of CONDUCT](CODE_OF_CONDUCT.md).
```

Reply:
this?

Reply:
Yeah, that should be it. 😄

Reply:
Oxford comma, just because grammarly suggested it doesn’t mean it’s actually needed. In this case the meaning is quite clear without it which is why I didn’t add it.

Reply:
For me, which sounds better than that in this context. Not sure why grammarly is flagging it, the checker I used doesn’t. Please feel free to explain the rule and why it should be that instead, otherwise I’ll leave it as it is.

Reply:
That confused me for a second, I was certain I added that comma 😂